<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReservWindow</class>
 <widget class="QDialog" name="ReservWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>224</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="le_fio">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_tel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_place">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lab_fio">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>ФИО</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_tel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Телефон</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_route">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Маршрут</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_place">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Место</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_price">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Оплата</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_enter">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>170</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Подтвердить</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_route">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Н. Новгород</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Иваново</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Кострома</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ярославль</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Сергиев Посад</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Москва</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Рязань</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Арзамас</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lab_price_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>350 руб</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
